### DISCUSSION TITLE
Unexpected Behavior when Using Translation Values for Sidebar with Tailwind CSS Classes

### DISCUSSION THREAD

**Discussion by naidaon** [tailwindlabs/tailwindcss#13053] [Help]
**What version of Tailwind CSS are you using?**

I am using Tailwind CSS v3.4.1.

**What build tool (or framework if it abstracts the build tool) are you using?**

I am using the Tailwind CLI to build an output file for use with a Flask HTML template.

**What version of Node.js are you using?**

I am using Node.js v20.11.0.

**What browser are you using?**

I am using Safari and Firefox.

**What operating system are you using?**

I am using macOS.

**Reproduction URL**

Here is the CodePen reproduction URL: [https://codepen.io/Denartha10/pen/NWmKVmV](https://codepen.io/Denartha10/pen/NWmKVmV)

**Describe your issue**

When attempting to toggle the visibility of a sidebar and shift the main content to the right using custom translation values with Tailwind CSS classes, unexpected behavior occurs. Specifically, when using the custom translation value `-translate-x-96` to shift the sidebar, the expected behavior is not observed. The sidebar does not move as expected, and the layout behaves unexpectedly.

**Steps to reproduce:**
1. Click on the toggle button to trigger the sidebar transition.
2. Observe the behavior of the sidebar and main content.

**Expected behavior:**
- The sidebar should smoothly transition out of view when toggled.
- The main content should shift to the right by a specified distance, maintaining a visually appealing layout.

**Actual behavior:**
- The sidebar behavior is unexpected, and it does not smoothly transition out of view as expected.
- The main content may not shift as intended, leading to layout inconsistencies.

**Additional notes:**
- This issue occurs across different browsers (Safari and Firefox) on macOS.
- The issue is reproducible using the provided CodePen URL.

**Comment by adamwathan** [comment]
Hey! This looks to me like it works when you update the HTML to correspond with the JS. You need to remove `-translate-x-full` from the sidebar in the HTML and replace it with `-translate-x-64` if that's the class you are toggling.

https://codepen.io/adamwathan/pen/gOyOYPX

**Reply by naidaon** [reply]
Haha, that feels like a noob mistake but thank you that really helps!
